common/test/common/HtmlCleanerTest.scala

```diff
@@ -0,0 +1,129 @@
+package common
+
+import org.jsoup.Jsoup
+import org.scalatest.{FlatSpec, Matchers}
+
+import pagepresser.{InteractiveImmersiveHtmlCleaner}
+
+class HtmlCleanerTest extends FlatSpec with Matchers {
+
+  "InteractiveImmersiveHtmlCleaner" should "only clean interactive immersives" in {
+    val doc = Jsoup.parse("""<html>
+              |<head>
+              |<link rel="canonical" href="https://www.theguardian.com/world/ng-interactive/2020/apr/08/coronavirus-100">
+              |</head>
+              |<body class="is-immersive is-immersive-interactive"></body>
+              |</html>""")
+
+    InteractiveImmersiveHtmlCleaner.canClean(doc) should be(true)
+  }
+
+ "Cleaned interactive immersives" should "have top banner ads removed" in {
+    val doc = Jsoup.parse("""<html>
+              |<body>
+              |<div id="bannerandheader">
+              |<div class="top-banner-ad-container js-top-banner"></div>
+              |<header></header>
+              |</div>
+              |</body>
+              |</html>""".stripMargin)
+
+    val want = Jsoup.parse("""<html>
+               |<body>
+               |<div id="bannerandheader">
+               |<header></header>
+               |</div>
+               |</body>
+               |</html>""".stripMargin).toString
```

Comment:
I had some issues around equality - I think it's because I couldn't easily compare jsoup Documents. I used toString so I could use the `equal` matcher successfully